pkg/registry/apis/provisioning/finalizers.go

```diff
@@ -0,0 +1,142 @@
+package provisioning
```

Comment:
Good call putting another file. I think that in the polishing phase we should move this and the controller into each one package.